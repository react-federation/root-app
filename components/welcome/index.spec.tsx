import { BrowserRouter } from 'react-router-dom'
import setup from 'utils/test/setup'
import Welcome from '.'

describe('Welcome', () => {
  it('should match snapshot', () => {
    const { asFragment } = setup(Welcome, {}, BrowserRouter)

    expect(asFragment()).toMatchSnapshot()
  })
})
