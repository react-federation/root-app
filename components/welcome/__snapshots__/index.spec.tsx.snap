// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 960px;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
}

.emotion-1 {
  padding-bottom: 2px;
}

.emotion-2 {
  display: inline-block;
  white-space: nowrap;
  vertical-align: middle;
  margin-right: 2px;
}

.emotion-3 {
  display: inline-block;
  white-space: nowrap;
  vertical-align: middle;
}

.emotion-4 {
  background-color: gray.50;
  padding: 4px;
  border-radius: md;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-5>*:not(style)~*:not(style) {
  margin-top: 0.5rem;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

<div
    class="emotion-0"
  >
    <div
      class="emotion-1"
    >
      <span
        class="chakra-badge emotion-2"
      >
        Getting started
      </span>
      <span
        class="chakra-badge emotion-3"
      >
        React ⚛️
      </span>
    </div>
    <div
      class="emotion-4"
    >
      <div
        class="chakra-stack emotion-5"
      >
        <h2
          class="chakra-heading emotion-6"
        >
          Welcome to your Root App!
        </h2>
        <p
          class="chakra-text emotion-6"
        >
          Here put your content
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
